.github/CODEOWNERS

```
@@ -0,0 +1 @@
+/.github/ @thomass-dev @rouk1 @augustebaum
```

I'm pretty sure that this is for security but could you quickly describe why this file is needed ?

Should we add the whole team ?

```suggestion
/.github/ @thomass-dev @rouk1 @augustebaum @probabl-ai/product 
```